Makefile

```diff
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.13
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.0
```

Comment:
CI test image will also need to be bumped
https://github.com/terraform-google-modules/terraform-google-project-factory/blob/87d2df092daa2ea5efd36a98fbbd946ce5722bf7/build/int.cloudbuild.yaml#L153